target/linux/realtek/dts-5.10/rtl838x.dtsi

```diff
@@ -60,25 +62,43 @@
 
 	compatible = "realtek,rtl838x-soc";
 
+	osc: oscillator {
+		compatible = "fixed-clock";
+		#clock-cells = <0>;
+		clock-frequency = <25000000>;
+	};
+
+	ccu: clock-controller {
+		compatible = "realtek,rtl8380-clock";
+		#clock-cells = <1>;
+		clocks = <&osc>;
+		clock-names = "ref_clk";
+		cpu-range = <375000000 550000000>;
+		mem-range = <300000000 350000000>;
+		lxb-range = <200000000 200000000>;
+	};
+
 	cpus {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		frequency = <500000000>;
 
 		cpu@0 {
 			compatible = "mips,mips4KEc";
 			reg = <0>;
+			clocks = <&ccu CLK_CPU>;
 		};
 	};
 
-	chosen {
-		bootargs = "console=ttyS0,115200";
+	sram0: sram@9f000000 {
+		compatible = "mmio-sram";
+		reg = <0x9f000000 0x10000>;
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges = <0 0x9f000000 0x10000>;
 	};
 
-	lx_clk: lx_clk {
-		compatible = "fixed-clock";
-		#clock-cells = <0>;
-		clock-frequency = <200000000>;
+	chosen {
+		bootargs = "console=ttyS0,115200";
```

Comment:
this shouldn't be here should it? For one, 'chosen' should be much higher (nodes are alphabetically sorted) but you might want to keep this in a generic commit

Reply:
this diff is STRANGE as I did not touch chosen. See a few linesa above where it was deleted.

Reply:
probably git being dumb. Just double check its sorted in the correct location; that might have triggered git to think you moved it.